configs/exp.yaml

```diff
@@ -0,0 +1,77 @@
+modules:
+  - seacliff
+  - galsim_extra
+
+image:
+  type: RubinCalexp
+
+  nobjects:
+    type: MixedNObjects
+    ngalaxies:
+        type: RandomPoisson
+        mean: 1000
+
+  world_pos:
+    type: RADec
+    type: RADec
+    ra:
+      type: Degrees
+      theta: { type: Random, min: "$ra_min_deg", max: "$ra_max_deg" }
+    dec:
+      type: Radians
+      theta:
+        type: RandomDistribution
+        function: "math.cos(x)"  # Uniform on the sky means P(delta) ~ cos(delta)
+        x_min: "$numpy.radians(dec_min_deg)"
+        x_max: "$numpy.radians(dec_max_deg)"
+
+psf:
+  type: RubinPSF
+
+gal:
+  type: Exponential
+  half_light_radius: 0.5
+  flux:
+    type: Eval
+    str: "10**(0.4*(30-mag))"
```

Comment:
Not sure where you're expecting to get mag from.  But right now, this won't work.  Probably should at least annotate a #TODO to say you need to figure out where the magnitudes come from.